src/test/test_subman.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+TMP=/tmp/subman$$
+mkdir $TMP
+trap "rm -fr $TMP" EXIT
+
+export PATH=$TMP:$PATH
+
+cat > $TMP/ceph-disk <<EOF
+echo '[{"partition":[{"type":"data","path":"/dev/foo/bar"}]}]'
+EOF
+chmod +x $TMP/ceph-disk
+
+cat > $TMP/df <<EOF
+echo Used
+echo $((2 * 1024 * 1024 * 1024))
+EOF
+chmod +x $TMP/df
+
+cat > $TMP/expected <<EOF
+{
+"band.storage.usage": 2
+}
+EOF
+export CEPH_FACTS_FILE=$TMP/facts
+script/subman
```

Comment:
might want to use `detect-build-env-vars.sh` to initialize the env vars, and use the absolute path instead.